26-map-filter-reduce/solution.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.map = function(arr, func) {
+  let newArr = [];
+  
+  for(let i = 0; i < arr.length; i++) {
+    newArr.push(func(arr[i]));
+  }
+
+  return newArr;  
+}
+
+exports.filter = function(arr, func) {
+  let newArr = [];
+
+  for(let i = 0; i < arr.length; i++) {
+    if(func(arr[i])) {
+      newArr.push(arr[i]);
+    }
+  }
+
+  return newArr;
+}
+
+exports.reduce = function(arr, func, startVal) {
```

Comment:
reduce should use the first value in the array if an initial value is not provided, how could you code that in?